share/p2p/peers/manager.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/celestiaorg/celestia-node/share/availability/discovery"
 )
 
+const (
+	ResultSuccess sampleResult = iota
+	ResultFail
+	ResultPeerMMisbehaved
```

Comment:
do we have to expose these consts?

Reply:
yes unless we make a shrexgetter package where both this and shrexgetter live